requirements.pip

```diff
@@ -5,5 +5,5 @@ protobuf
 pytest
 six
 sympy
-tensorflow >= 1.5.0, <=1.15.3
+tensorflow >= 1.5.0, <=1.14
```

Comment:
Whats the reason to change this?

Reply:
There are 4 or 5 batch norm related tests that are failing. 

Developer can still 1.15 and most of their models would still convert. But 1.14 seems to be declaring the tested version.

Reply:
In that case its okay to change here, but don't change it in the `setup.py` file, because that is what is used by pip while installing the package, if you use 1.14 there and the user downloads tfcoreml from pypi in an environment which has TF 1.15, it might not install it, or it might give a warning (you can verify it locally by creating the wheel and installing it)

Reply:
In such a case, it automatically uninstalls TF 1.15 and installs TF 1.14.

Reply:
Yeah, thats not what we want, do we?

Reply:
In my opinion, it's totally fine because:

1. Users for whom it doesn't matter whether its TF 1.15 or TF 1.14, we are good. Things would work fine as it is.
2. And for users, who have a hard requirement to use TF 1.15, they can re-install TF 1.15, everything will work fine except that now they would get a warning which they can already reason about.